hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestMergeIntoTable.scala

```diff
@@ -115,6 +116,65 @@ class TestMergeIntoTable extends HoodieSparkSqlTestBase with ScalaAssertionSuppo
     })
   }
 
+  test("Test MergeInto with more than once update actions") {
+    withRecordType()(withTempDir {tmp =>
+      val conf = new SparkConf().setAppName("insertDatasToHudi").setMaster("local[*]")
+      val spark = SparkSession.builder().config(conf)
+        .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
```

Comment:
Both `conf` and `spark` can be removed, both in the `HoodieSparkSqlTestBase` class.

Reply:
ok，thanks. I add a Todo issue https://issues.apache.org/jira/browse/HUDI-5918 @XuQianJin-Stars